crates/core/component/dex/src/swap_claim/proof.rs

```diff
@@ -437,14 +437,61 @@ mod tests {
     use proptest::prelude::*;
     use rand_core::OsRng;
 
+    #[derive(Debug)]
+    struct TestBatchSwapOutputData {
+        delta_1: Amount,
+        delta_2: Amount,
+        lambda_1: Amount,
+        lambda_2: Amount,
+        unfilled_1: Amount,
+        unfilled_2: Amount,
+    }
+
+    fn filled_bsod_strategy() -> BoxedStrategy<TestBatchSwapOutputData> {
+        let delta_1 = (4001..2000000000u128).prop_map(Amount::from);
+        let delta_2 = (4001..2000000000u128).prop_map(Amount::from);
```

Comment:
Do we need to have such a large range?

Reply:
That's not that big if you account for the denom scaling factors, typically 10^6, 10^9 or 10^18 (!)